app/controllers/stats_controller.rb

```diff
@@ -97,7 +97,7 @@ def generate_incoming_new_words_report
   end
 
   def translator_report
-    @languages = Project.pluck(:targeted_rfc5646_locales, :base_rfc5646_locale).map { |hash, base| hash.keys - [base]}.flatten.uniq.sort
+    @languages = Project.pluck(:targeted_rfc5646_locales, :base_rfc5646_locale).map { |hash, base| hash.keys - [base]}.flatten.map(&:downcase).uniq.sort
```

Comment:
We have ran into issues with locale comparison in house recently. Lower case all locale here will not provide additional help here.

Reply:
On the report, they are uppercase. I'm using `downcase` here solely for comparison. Some locales are mixed case, some upper, some lower.

Reply:
Please original local string, no upcase/downcase. These strings are used to filter the database later. Downcasing the locale will not return the expect result while the database tables/columns are configured as case sensitive.

We already ran into the issue in other places. I will address the inconsistent locale in other PR soon.

You can upcase the locale for the CSV only, if needed.

Reply:
Please update the PR so we can approve and merge. Thanks

Reply:
I understand. Is it ok that there will be duplicates in the list of options that the user can select? WIthout normalizing the case, this is the list you get back:
```
"EN", "EN-AU", "EN-CA", "EN-GB", "ES", "FR", "FR-CA", "JA", "en-AU", "en-CA", "en-GB", "en-IN", "en-au", "en-ca", "en-gb", "es", "es-MX", "es-US", "es-us", "fr", "fr-CA", "fr-ca", "ja"```

You can see Japan is in there twice, as is Australia, and Great Britain is in there three times, to name a few. As far as I can tell, these aren't distinct locales and it's possible to exclude one but not the other. It may be confusing for users. That said, I will check in a change with this reverted in a minute.